framework/include/bcs/HFEMDirichletBC.h

```diff
@@ -0,0 +1,35 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
```

Comment:
Possibly we will get documentation errors because we are missing few markdown files for the newly added dg kernels and bcs.